### ISSUE TITLE
I tried to reproduce Wide-DenseNet-BC results on cifar10, but got 0.5% more than your error

### ISSUE THREAD

**Issue by seasonyc** [liuzhuang13/DenseNet#49]
I tested Wide-DenseNet-BC (L=40, k=48) on CIFAR-10 augmentation, see https://github.com/seasonyc/densenet/blob/bf99d7f459ca7754c37ff58c6610eb76e93f7990/cifar10-test.py#L217 in https://github.com/seasonyc/densenet
but could only get 4.5% error rate.

I tried to tune some hyper parameters, e.g. dropout, weight decay, learning rate... but always couldn't get better result. Now I am testing to follow the lr decay of wide resnet training, i.e.  initial 0.1, by 0.2 per 60 epochs, but I very suspect if it will take effect...

Would you like to give me any suggestion for it?

Thanks
YC

**Comment by liuzhuang13** [comment]
One difference I noted is that you normalize the input by dividing 255, but we normalize the input by channel means and stds, that might be a difference. 

https://github.com/liuzhuang13/DenseNet/blob/master/datasets/cifar10.lua#L37

**Comment by taineleau** [comment]
Please also check https://github.com/felixgwu/img_classification_pk_pytorch/blob/master/models/densenet.py if you'd like to reproduce DenseNet's result in PyTorch.

**Comment by liuzhuang13** [comment]
Besides, the data augmentation scheme seems to be significantly different. 
![image](https://user-images.githubusercontent.com/8370623/43027889-788ed698-8c31-11e8-8810-18db99fab64b.png)
![image](https://user-images.githubusercontent.com/8370623/43027915-978d8cb0-8c31-11e8-9038-8c051e32b892.png)

I think you can refer to this code for data augmentation in keras,
https://github.com/robertomest/convnet-study/blob/master/train.py

**Comment by liuzhuang13** [comment]
I noted the data augmentation in your code is significantly different, I think you can refer to this code for data augmentation in keras, https://github.com/robertomest/convnet-study/blob/master/train.py

**Comment by seasonyc** [comment]
I updated https://github.com/seasonyc/densenet/blob/master/cifar10-test.py
at https://github.com/seasonyc/densenet/blob/571b0d6d3873fbec3eec9e631370e74172d05a4b/cifar10-test.py#L51
and https://github.com/seasonyc/densenet/blob/571b0d6d3873fbec3eec9e631370e74172d05a4b/cifar10-test.py#L141

The validation acc has some promotion, now the top acc is 95.64%.  I think it's acceptable although it still has a bit gap than yours.

Thanks
YC
